.github/workflows/releaser-master.yml

```diff
@@ -21,8 +24,25 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0
+
+      - name: Determine release mode
+        id: release-mode
+        run: |
+          if [[ "${{ github.event_name }}" == "pull_request" ]]; then
+            echo "mode=test" >> $GITHUB_OUTPUT
+            echo "args=build --snapshot --skip=before" >> $GITHUB_OUTPUT
+          else
+            echo "mode=release" >> $GITHUB_OUTPUT
+            echo "args=release --clean --snapshot" >> $GITHUB_OUTPUT
```

Comment:
release --clean --nightly was used for a reason.

What we are currently doing is a release not from a tag but from a branch although it is master.
AFAIK the only way to release and perform all the steps avoiding to create a tag is using --nightly.

release --clean --snapshot will not push the docker images